MdeModulePkg/Bus/Pci/PciBusDxe/PciBus.h

```diff
@@ -284,6 +284,8 @@ struct _PCI_IO_DEVICE {
   UINT16                                       BridgeIoAlignment;
   UINT32                                       ResizableBarOffset;
   UINT32                                       ResizableBarNumber;
+  BOOLEAN                                      IgnoreROM;      // MS_CHANGE

```

Comment:
Recommend making these `MU_CHANGE` throughout code modifications for consistency.

Reply:
Removed.